query-specifications/bi-read-19.yaml

```diff
@@ -5,9 +5,13 @@ title: Stranger's interaction
 description: |
   For all the *Persons* born after a certain `date`, find all the strangers they interacted with, where strangers are *Persons* that do not *know* each other. There is no restriction on the date that strangers were born.
 
-  Consider only strangers that are members of *Forums* tagged with a *Tag* of `tagClass1` (direct children not transitive) AND members of *Forums* tagged with a *Tag* of `tagClass2` (direct children not transitive). The tags may be attached to the same *Forum* or they may be attached to different *Forums*.
+  Consider only strangers that are
```

Comment:
This card is not being rendered well on the PDF. The bullet points do not appear